apps/sms/services/js/message_manager.js

```diff
@@ -471,8 +471,9 @@ var MessageManager = {
   },
 
   /**
-   * Deletes messages using the message id provided.
-   * @param {number} id The ID of the message to be deleted.
+   * Deletes messages using the parameter provided.
+   * @param {number|array|object} id <long | MmsMessage | SmsMessage and
```

Comment:
Maybe something like this would be better:
```
@param {number|SmsMessage|MmmsMessage|Array<number|SmsMessage|MmsMessage>} id
Message id, message instance or list of aforementioned entities used to identify message(s) to delete.
```